contrib/travis/travis-build-linux.sh

```diff
@@ -1,23 +1,24 @@
 #!/bin/bash
-set -e
-
-if [[ $TRAVIS_PYTHON_VERSION != 3.4 ]]; then
-  exit 0
-fi
+set -ev
 
 if [[ -z $TRAVIS_TAG ]]; then
-  exit 0
+  echo TRAVIS_TAG unset, exiting
+  exit 1
 fi
 
+BUILD_REPO_URL=https://github.com/AXErunners/electrum-axe.git
+
 cd build
 
-BUILD_REPO_URL=https://github.com/axerunners/electrum-axe.git
 git clone --branch $TRAVIS_TAG $BUILD_REPO_URL electrum-axe
 
 docker run --rm \
     -v $(pwd):/opt \
     -w /opt/electrum-axe \
-    -t zebralucky/electrum-axe-winebuild:Linux /opt/build_linux.sh
```

Comment:
https://hub.docker.com/r/zebralucky/electrum-dash-winebuild/

have a look at this